index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
+  <title>Awesome books</title>
+  <link rel="stylesheet" href="./main.css">
+</head>
+<body>
+  
+  <nav class="nav-bar">
+    <p id="logo">Awesome-Books</p>
+    <ul class="nav-list">
+      <li class="nav-item" id="list">List </li>
+      <li>|</li>
+      <li class="nav-item" id="addNew">Add new</li>
+      <li>|</li>
+      <li class="nav-item" id="contact">Contact</li>
+    </ul>
+  </nav>
+<section class="wrapper">
+  <p id="time"></p>
+  <div id="homePage">
+  <h1>Welcome to Awesome-Books!</h1>
+  </div>
+
+  <div id="listSection">
+    <h1>All Awesome Books</h1>
+    <div class="bookShelf" id="bookShelf"></div>
+  </div>
```

Comment:
- [ ] **[OPTIONAL]** Please, consider displaying the list of books on the welcome page. I mean when the user reloads the page, make sure the list of books is already displayed without the need to click on list book button. This is not a project requirement but it will avoid confusion and highly improve the users experience 😄 📱